fluster/test_suite.py

```diff
@@ -156,6 +164,8 @@ def from_json_file(cls: Type["TestSuite"], filename: str, resources_dir: str) ->
                 data["failing_test_vectors"] = dict(map(TestVector.from_json, data["failing_test_vectors"]))
             data["test_vectors"] = dict(map(TestVector.from_json, data["test_vectors"]))
             data["codec"] = Codec(data["codec"])
+            if "test_method" in data and data["test_method"] is not None:
+                data["test_method"] = TestMethod(data["test_method"])
```

Comment:
"test_method" should never be None. it's either present with a value or not present and we default to MD5